github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml

```diff
@@ -351,6 +353,10 @@ branch-protection:
           branches:
             main:
               protect: true
+        common-instancetypes:
+          branches:
+            master:
```

Comment:
Are you sure that the branch is still called `master`?

Reply:
Ah apologies, nice catch!